src/schema/v2/me/index.ts

```diff
@@ -239,7 +244,7 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
     otherRelevantPositions: {
       type: GraphQLString,
       description: "Collector's position with relevant institutions",
-      resolve: ({ other_relevant_positions }) => other_relevant_positions,
+      resolve: collectorProfileResolver("other_relevant_positions"),
```

Comment:
`other_relevant_positions` is not exposed from the `meLoader` but from the `collectorProfileLoader`.